pkgs/development/compilers/julia/stable.nix

```diff
@@ -129,6 +123,8 @@ stdenv.mkDerivation rec {
     export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}
   '';
 
+  enableParallelBuilding = true;
```

Comment:
Already enforce by cmake.

Reply:
Despite `dontUseCMakeConfigure`? Isn't that a bug, and even if not, do we want to rely on it?

Reply:
@7c6f434c is correct here, you have to force parallel builds like this as Julia uses GNU Make that then passes on the desire to use parallel builds when building (a subset of) its dependencies that uses CMake. Happy if there is another way to declare it, but removing it will lead to non-parallel builds as noted in the commit that re-introduced `enableParallelBuilding = true`.

Reply:
> Despite dontUseCMakeConfigure? 

Apparently not https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/build-managers/cmake/setup-hook.sh#L125